journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
+liberty to go to their friends in town at night,
+and to return in the morning;
+and if he wasn`'t willing to grant them a little straw,
+then to let them go lie in their own beds; which he sullenly denied,
 calling them a company of rogues and knaves, and such like terms.
-He asked me, what made me plead for them; I told him, they were my friends.
-He answered, Why your friends?
-You are no Quaker, are you?
-I said, I am called a Quaker.
-He answered, You do not look like a Quaker; and he looked me in my face,
-and on my hands and body.
+He asked me what compelled me to plead for them. I told him they were my friends.
+He answered, "`Why your friends? You are no Quaker, are you?`"
+I said, "`I am called a Quaker.`"
+He answered, "`You do not look like a Quaker;`" and he looked me in the face,
+and at my hands and body.
 I desired him not to disgrace me so, as to tell me I was no Quaker.
-Then he asked me, where I lived; I told him, when I was at home, I lived at Welch-pool,
-and my family was there.
-But said he, where are you now?
-I asked him, whether he did not know I was a prisoner there with my friends;
-and he asked me, whether I did lie upon the boards with them; I told him I did.
-He said, he was sorry for it; but went away in anger, being much discontented in himself.
-
-I did not see him till next morning, at which time I went to him again,
-and discoursed friendly with him; he said, he enquired about me in town,
-and I might take the liberty of the town.
-I acknowledged his kindness; but told him, it would be no comfort to me,
-to have the liberty of the town, and leave my friends and brethren there.
-He said, then I might stay there with them.
-So I did not see him till the next morning and then I went to him again.
-He was so cross and ill-conditioned, he would not allow any other Friend to speak to him.
-William Gibson did so judge him for his wickedness,
-that he kept him close in a room by himself.
-After five or six nights lying on the boards, I prevailed with him,
-that Friends might have the liberty of the town in the night,
-and be there in the morning.
+Then he asked me where I lived; I told him that when I was at home,
+I lived at Welch-pool, and my family was there.
+"`But,`" said he, "`where are you now?`"
+I asked him whether he knew that I also was a prisoner there with my friends;
+and he asked me whether I slept upon the boards with them; I told him I did.
+He said he was sorry for it, but went away in anger, being much discontented in himself.
+
+I did not see him again till next morning, at which time I went to him again,
+and discoursed kindly with him. He said he had enquired about me in town,
+and that I might have the liberty of the town if I wished.
+I acknowledged his kindness, but told him it would be no comfort to me
+to have the liberty of the town, and leave my friends and brethren there in prison.
+He said I might stay there in prison with them, then.
```

Comment:
I'm not sure the ", then" at the end works. Maybe we can change it to a quotation, and say: He said, "Then you may stay there in the prison with them."